File: Tatarstan/Bars.GkhGji.Regions.Tatarstan/libs/B4/model/actactionisolated/Definition.js
Ext.define('B4.model.actactionisolated.Definition', {
    extend: 'B4.base.Model',
    proxy: {
        type: 'b4proxy',
        controllerName: 'ActActionIsolatedDefinition'
    },
    fields: [
        {name: 'Number'},
        {name: 'Date'},
        {name: 'DefinitionType'},
        {name: 'Official'},
        {name: 'ExecutionDate'},
        {name: 'RealityObject'},
        {name: 'Description'},
        {name: 'Act'}
    ]
});

File: Bars.GkhGji.Contracts/Meta/Definition.cs
namespace Bars.GkhGji.Contracts.Meta
{
    using System;
    using Bars.B4.Utils;

    public class Definition
    {
        [Display("Номер")]
        public string Number { get; set; }

        [Display("Дата")]
        public DateTime Date { get; set; }

        [Display("Тип определения")]
        public string DefinitionType { get; set; }

        [Display("ДЛ, вынесшее решение")]
        public Inspector Official { get; set; }

        [Display("Дата исполнения")]
        public DateTime ExecutionDate { get; set; }

        [Display("Описание")]
        public string Description { get; set; }
    }
}
